nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -695,4 +695,6 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `grafana-loki` package was updated to 3.0.0 which includes [breaking changes](https://github.com/grafana/loki/releases/tag/v3.0.0)
 
+- `drbd` utils have been bumped to version 9.28.0
```

Comment:
We should update the above release notes instead.

Reply:
It is for different packages (utils and the kernel driver), but let's combine the message...